### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by sbonnalc** [nats-io/k8s#424] [Q&A] [answered]
Hello,
We're using NATS, and also the nats-kafka and nack.
For benchmark purpose we ensure placement with affinity and tolerations in order to run on dedicated nodes.
nats-kafka and nack doesn't have yet those affinity and tolerations blocks allowed in HELM values.yaml.
That's why I opened PR https://github.com/nats-io/k8s/pull/420
Is that the right way to contribute here?

**Comment by wallyqs** [comment, marked as answer]
yes this is the right place, I've just merged your PR. Thanks!

**Reply by sbonnalc** [reply]
Thanks to you all :-)
